### PR TITLE
PMM-12792 Relelase Branch 2.41.1

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

File: variables.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # PMM Version for HTML
 # See also mkdocs.yml plugins.with-pdf.cover_subtitle and output_path
 
-release: '2.41.0'
-version: '2.41.0'
-release_date: 2023-12-12
+release: '2.41.1'
+version: '2.41.1'
+release_date: 2024-01-09
```
Comment (rnovikovP) [inline variables.yml:6]:
```suggestion
release_date: 2024-01-24
```

**Review by rnovikovP — APPROVED** [review]

**Comment by ondrejpatocka** [comment]
Hi, I noticed there are old Jira issues links in RN on
https://pmm-doc-pr-1188.onrender.com/release-notes/2.41.1.html#improvements
Is it expected?

File: docs/release-notes/2.41.1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+We've made it easier to add and monitor MongoDB Arbiter nodes in PMM, as this no longer requires workarounds for cases where authorization is enabled on the MongoDB replica set. Additionally, Arbiter nodes are now displayed correctly on the dashboards:
+
+![!](../_images/Arbiter.png)
+
+### New metric for sharded MongoDB insights
+
+We've added a new metric to our MongoDB exporters: `mongodb_shards_collection_chunks_count`. This metric is available for sharded Mongo instances, offering insights into the distribution of collection chunks across different shards.
+
+Explore and leverage this new metric to enhance your understanding of your sharded MongoDB setup.
```
Comment (rnovikovP) [inline docs/release-notes/2.41.1.md:24]:
```suggestion
This version is not yet used in any dashboard, but you can use it in your custom dashboards (you can use https://www.youtube.com/watch?v=vk1QBiMVzz4 and https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-manage-dashboards.html )

Please share your feedback on our forum (forum.percona.com)
```

File: docs/release-notes/2.41.1.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
```
Comment (rnovikovP) [inline docs/release-notes/2.41.1.md:25]:
```suggestion

### New Experimental MongoDB Instance Summary dashboard
We have introduced a new MongoDB Instance summary in this release that offers PMM users a simplified view of the most critical MongoDB metrics. You can locate it in the "Experimental" folder of your PMM installation. We request you to verify it against your MongoDB instance and share your feedback on our forum.

```

**Comment by sleto-it** [comment]
Direct link to preview the changes, as the 2.41.1.md page is not currently linked in the release notes index:

https://pmm-release-branch-pr-1188.onrender.com/release-notes/2.41.1.html

**Comment by sleto-it** [comment]
[PMM-12350](https://perconadev.atlassian.net/browse/PMM-12350) is not mentioned in the release notes - but I assume it was fixed as well?

**Comment by ondrejpatocka** [comment]
Hi @rasika-chivate and @catalinaadam I'm checking latest RN for 2.41.1 on https://pmm-doc-pr-1188.onrender.com/release-notes/2.41.1.html
and there are bugs (and improvements) missing/not mentioned, compared to Jira tickets list below.
Is it intentional, or am I checking wrong source?

# Release notes - Percona Monitoring and Management - 2.41.1

### Improvement

[PMM-12390](https://perconadev.atlassian.net/browse/PMM-12390) MongoDB Backup / Service Name - Make the text search case insensitive

[PMM-12510](https://perconadev.atlassian.net/browse/PMM-12510) No support of MongoDB arbiters with Authorization enabled

[PMM-12712](https://perconadev.atlassian.net/browse/PMM-12712) Collect Data about Sharded collections in MongoDB 

[PMM-12750](https://perconadev.atlassian.net/browse/PMM-12750) \[Tech Preview\] Single dashboard with important MongoDB KPIs

[PMM-12846](https://perconadev.atlassian.net/browse/PMM-12846) Telemetry: data for the client node version 

[PMM-12866](https://perconadev.atlassian.net/browse/PMM-12866) K8s experimental dashboard update 

### Bug

[PMM-9407](https://perconadev.atlassian.net/browse/PMM-9407) PMM2-Client Debian doesn't include queries.yaml inside Postgresql Medium Resolution

[PMM-9825](https://perconadev.atlassian.net/browse/PMM-9825) QAN: Strange values in User Name filter with pg\_stat\_monitor

[PMM-12290](https://perconadev.atlassian.net/browse/PMM-12290) Comments are parsed by default for remotely added PostgreSQL and MySQL

[PMM-12350](https://perconadev.atlassian.net/browse/PMM-12350) Error in mongod\_exporter logs when pmm-agent is connected to mongos

[PMM-12425](https://perconadev.atlassian.net/browse/PMM-12425) QAN picking different date range will cause an error 

[PMM-12473](https://perconadev.atlassian.net/browse/PMM-12473) Unable to add external service with a query string in the metrics-path

[PMM-12621](https://perconadev.atlassian.net/browse/PMM-12621) Show all service types in anonymous mode

[PMM-12652](https://perconadev.atlassian.net/browse/PMM-12652) Issues upgrading to >=2.38.x with SELinux enabled

[PMM-12738](https://perconadev.atlassian.net/browse/PMM-12738) certificate.conf required, but not mentioned anywhere in helm charts

[PMM-12781](https://perconadev.atlassian.net/browse/PMM-12781) permission denied for function pg\_ls\_waldir

[PMM-12807](https://perconadev.atlassian.net/browse/PMM-12807) "Skip TLS certificate and hostname validation" flag isn't working.

[PMM-12815](https://perconadev.atlassian.net/browse/PMM-12815) pmm client fails to add mongoDB via socket

[PMM-12816](https://perconadev.atlassian.net/browse/PMM-12816) PMM Upgrade sporadically fails for 2.37.1 and 2.38.0 to 2.41.1

**Comment by catalinaadam** [comment]
> Hi @rasika-chivate and @catalinaadam, I noticed there are old Jira issues links in RN on https://pmm-doc-pr-1188.onrender.com/release-notes/2.41.1.html#improvements Is it expected? For now the old->new Jira links redirect works, but will be disabled eventually.
@ondrejpatocka, for PXB 8.2 they got mapped automatically  and opened in Cloud Jira

File: docs/release-notes/2.41.1.md (hunk 3 above)
Comment (rnovikovP) [reply docs/release-notes/2.41.1.md:29]: @catalinaadam  pls 👁️  this is for the [PMM-12750](https://perconadev.atlassian.net/browse/PMM-12750)

File: docs/release-notes/2.41.1.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+### New experimental MongoDB Instance Summary dashboard
+
+This release also introduces a new MongoDB dashboard that offers a simplified view of the most critical MongoDB metrics. You can find the new **MongoDB Instance Summary** dashboard in the **Experimental** folder of your PMM installation. We encourage you to evaluate its performance with your MongoDB instance and share your valuable feedback on our forum.
+
+## Improvements
+
+- [PMM-12390](https://jira.percona.com/browse/PMM-12390) - [Backup Management]: The **Service name** field on the Create Backup pages is now case-insensitive, allowing for more comprehensive and accurate results retrieval.
+- [PMM-12712](https://jira.percona.com/browse/PMM-12712) - MongoDB sharded cluster metrics now include a new metric to represent database and collection distribution across different shards. This helps gain insights into the overall balance of shards within MongoDB setups.
+- [PMM-12510](https://jira.percona.com/browse/PMM-12510) - Improved workflow for adding MongoDB Arbiter nodes to PMM, which also ensures that Arbiter nodes are now displayed correctly on the dashboards.
+
```
Comment (rnovikovP) [inline docs/release-notes/2.41.1.md:35]:
```suggestion

[PMM-12750](https://perconadev.atlassian.net/browse/PMM-12750) [Dashboards] Tech Preview* of the new MongoDB Instance Summary

[PMM-12866](https://perconadev.atlassian.net/browse/PMM-12866) [Dashboards] update of K8s experimental dashboard (Tech Preview*)

```

File: docs/release-notes/2.41.1.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## Bugs Fixed
+
```
Comment (rnovikovP) [inline docs/release-notes/2.41.1.md:37]: - [PMM-9407] (https://perconadev.atlassian.net/browse/PMM-9407): Fixed issue with executing Custom Queries for PostgreSQL on Debian due to the absence of queries.yaml within packages.
- [PMM-12290](https://perconadev.atlassian.net/browse/PMM-12290) [QAN] Fixed bugs with SQL comments parsed in queries when they were not intentionally enabled for remotely added PostgreSQL and MySQL.
- [PMM-12350](https://perconadev.atlassian.net/browse/PMM-12350) Fixed problem with mongod_exporter flood logs when connected to Mongos. 
- [PMM-12738](https://perconadev.atlassian.net/browse/PMM-12738) Fixed bug with inability to run PMM using Helm with customer certificates.  
- [PMM-12781](https://perconadev.atlassian.net/browse/PMM-12781) Fixed a permission error that flooded PostgreSQL logs by fixing the pg_ls_waldir function's execution permissions.

**Review by rnovikovP — CHANGES_REQUESTED** [review]

**Review by sleto-it — APPROVED** [review]
Approving from my side, as my feedback wrt PMM-12350 has been handled.

We still need to link page 2.41.1.md into the release note index, it seems.

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**